recipes/hyperbox-brain/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "hyperbox-brain" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 41ba26a8b285f48ef5f8e76c6f60fdab43b9ed689b7f2c531e7ade4442a83437
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+    - numpy
+    - wheel
+    - scikit-learn
+    - pandas
+    - matplotlib-base
+    - joblib
+    - scipy
+    - threadpoolctl
+    - plotly
```

Comment:
OPTIONAL: Normally you shouldn't need all the package requirements in this section. It is because your package imports itself in setup.py.

The recommended way for a package to get it's version during setup without importing the package is by using [something like this in](https://setuptools.pypa.io/en/latest/userguide/declarative_config.html) the setup.cfg:

```
[metadata]
name = my_package
version = attr: my_package.VERSION
```

Or using a tool like https://pypi.org/project/setuptools-scm/

Reply:
@carterbox Thanks for your comments. I have removed some unnecessary packages there. However, I have to keep some essential packages because in the setup file, I have the code to check the version of essential tools before installing our library.

Reply:
You shouldn't need to manually check if the dependencies are up to date. That is the job of the package manager (e.g. pip, poetry, conda). Just keep the dependency versions up to date in your package's metadata. i.e. the `install_requires` parameter.